src/seo/utils/createAlternateLanguageLink.ts

```diff
@@ -0,0 +1,18 @@
+import type { LocaleObject } from '../../intl';
+
+export function createAlternateLanguageLink(
```

Comment:
This method seems redundant. You can just put the logic in the `document` component

Reply:
This function is use in two different places
<img width="344" alt="image" src="https://user-images.githubusercontent.com/3620639/216579865-6ece3fce-2e45-4266-8f19-30e3b60a807b.png">